integration_tests/tests/athena-query-tests.ts

```diff
@@ -1,58 +1,35 @@
-import { snsValidEventPayload } from "../../src/handlers/int-test-support/helpers/payloadHelper";
+import { validEventPayload } from "../../src/handlers/int-test-support/helpers/payloadHelper";
 import { resourcePrefix } from "../../src/handlers/int-test-support/helpers/envHelper";
-import { publishToTestTopic } from "../../src/handlers/int-test-support/helpers/snsHelper";
-import { listS3Objects } from "../../src/handlers/int-test-support/helpers/s3Helper";
-import { poll } from "../../src/handlers/int-test-support/helpers/commonHelpers";
 import {
   startQueryExecutionCommand,
   waitAndGetQueryResults,
 } from "../../src/handlers/int-test-support/helpers/athenaHelper";
+import { generateEventViaFilterLambdaAndCheckEventInS3Bucket } from "../../src/handlers/int-test-support/helpers/testDataHelper";
 
 const prefix = resourcePrefix();
-const objectsPrefix = "btm_transactions";
 
-describe("\nPublish valid sns message and execute athena query\n", () => {
+describe("\nGenerate valid event and execute athena query\n", () => {
   beforeAll(async () => {
-    await publishToTestTopic(snsValidEventPayload);
-    const checkEventId = async (): Promise<boolean> => {
-      const result = await listS3Objects({
-        bucketName: `${prefix}-storage`,
-        prefix: objectsPrefix,
-      });
-      if (result.Contents !== undefined) {
-        return JSON.stringify(result.Contents.map((data) => data.Key)).includes(
-          snsValidEventPayload.event_id
-        );
-      } else {
-        return false;
-      }
-    };
-    const eventIdExists = await poll(
-      checkEventId,
-      (result: boolean) => result,
-      {
-        nonCompleteErrorMessage:
-          "Event Id check was not successful within the specified timeout",
-      }
+    await generateEventViaFilterLambdaAndCheckEventInS3Bucket(
+      validEventPayload
     );
-    expect(eventIdExists).toBeTruthy();
   });
 
   test("should contain eventId in the generated query results", async () => {
     const databaseName = `${prefix}-calculations`;
-    const queryString = `SELECT * FROM "btm_transactions_standardised" where event_id='${snsValidEventPayload.event_id}'`;
+    const queryString = `SELECT * FROM "btm_transactions_standardised" where event_id='${validEventPayload.event_id}'`;
     const queryId = await startQueryExecutionCommand({
       databaseName,
       queryString,
     });
     const queryResult = await waitAndGetQueryResults(queryId);
     expect(JSON.stringify(queryResult?.ResultSet?.Rows)).toContain(
-      snsValidEventPayload.event_id
+      validEventPayload.event_id
     );
   });
 });
 
-describe("\nPublish invalid sns message and execute athena query\n", () => {
+describe("\nGenerate invalid event and execute athena query\n", () => {
```

Comment:
In the test above, you use `beforeAll` and `generateEventViaFilterLambdaAndCheckEventInS3Bucket` with a `validEventPayload`. 
Maybe we can do something similar here and generate an event with an invalid payload?

```suggestion
describe("\nGenerate valid event and execute athena query\n", () => {
  test("should contain eventId in the generated query results", async () => {
    const { eventId } =
      await generateEventViaFilterLambdaAndCheckEventInS3Bucket(
        validEventPayload
      );

    const databaseName = `${prefix}-calculations`;
    const queryString = `SELECT * FROM "btm_transactions_standardised" where event_id='${eventId}'`;
    const queryId = await startQueryExecutionCommand({
      databaseName,
      queryString,
    });
    const queryResult = await waitAndGetQueryResults(queryId);
    expect(JSON.stringify(queryResult?.ResultSet?.Rows)).toContain(eventId);
  });
});

describe("\nGenerate invalid event and execute athena query\n", () => {
  test("should not contain eventId in the generated query results", async () => {
    const { eventId } =
      await generateEventViaFilterLambdaAndCheckEventInS3Bucket(
        invalidEventPayloadEventName
      );
    const databaseName = `${prefix}-calculations`;
    const queryString = `SELECT * FROM "btm_transactions_standardised" where event_id='${eventId}'`;
    const queryId = await startQueryExecutionCommand({
      databaseName,
      queryString,
    });
    const queryResult = await waitAndGetQueryResults(queryId);
    expect(JSON.stringify(queryResult?.ResultSet?.Rows)).not.toContain(eventId);
  });
});
```

Reply:
done it make sense to return the eventId from generateEventViaFilterLambdaAndCheckEventInS3Bucket